cli/azd/docs/alpha-features.md

```diff
@@ -0,0 +1,45 @@
+# Alpha Features
+
+The Azure Developer CLI includes a `alpha feature toggle` which allows the cli to enable `experimental`. This document provides the feature design for it.
+
+## Main requirement
+
+As a customer, I can explicitly ask `azd` to unblock and show any _alpha_ functionality. The features becoming available are not finalized, so they can be changed or removed in future releases.
+
+The strategy to ask `azd` to `toggle` between modes is:
+
+- Enable _all_ experimental features: `azd config set experimental.all on`
+- Enable _specific_ experimental feature: `azd config set experimental.feature on`
+- Disable _all_ experimental features: `azd config unset experimental.all`
+- Disable _specific_ experimental feature: `azd config unset experimental.feature`
```

Comment:
Is `experimental` still the root? or is it `alpha` now? Looks like the code is looking for `alpha` ?